backend/app.js

```diff
@@ -77,6 +77,21 @@ app.post("/api/googlesearch", async (req, res) => {
   }
 });
 
+// Fanout search route.
+app.post("/api/googlefanout", async (req, res) => {
+  try {
+    const question = req.body.text;
+    const response = await gameSearch.fanoutSearches({
```

Comment:
Given that this fanout query is relatively expensive (1 model call + 5 sapi calls), we will add a stub response so that an FE developer could receive the stubbed data instead of the real data while testing UIs. This should be made applicable to all other endpoints later. For now, I'm setting up for just this relatively complex and expensive flow.